python/nano/notebooks/tensorflow/cifar10/nano_tensorflow_fit_example.ipynb

```diff
@@ -0,0 +1,1054 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "10284b3f",
+   "metadata": {},
+   "source": [
+    "## BigDL-Nano Resnet example on CIFAR10 dataset\n",
+    "---\n",
+    "This example illustrates how to apply bigdl-nano optimizations on a image recognition case based on Tensorflow Keras framework. The basic image recognition module is implemented with tf.keras and trained on [CIFAR10](https://www.cs.toronto.edu/~kriz/cifar.html) image recognition Benchmark dataset. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "52401226",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from time import time\n",
+    "\n",
+    "import tensorflow as tf\n",
+    "from tensorflow import keras\n",
+    "from tensorflow.keras import layers\n",
+    "from tensorflow.keras.callbacks import ModelCheckpoint\n",
+    "from bigdl.nano.tf.keras import Model, Sequential"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "732c132d",
+   "metadata": {},
+   "source": [
+    "### CIFAR10 Data Module\n",
+    "---\n",
+    "Import the existing data module from keras.datasets and Normalize the images.\n",
+    "You could access [CIFAR10](https://www.cs.toronto.edu/~kriz/cifar.html) for a view of the whole dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "01236c02",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cifar10 = keras.datasets.cifar10\n",
+    "(train_images, train_labels), (test_images, test_labels) = cifar10.load_data()\n",
+    "\n",
+    "train_images = train_images.astype('float32') / 255.0\n",
+    "test_images = test_images.astype('float32') / 255.0\n",
+    "\n",
+    "# Convert class vectors to binary class matrices.\n",
+    "train_labels = keras.utils.to_categorical(train_labels, 10)\n",
+    "test_labels = keras.utils.to_categorical(test_labels, 10)\n",
+    "\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "train_images, val_images, train_labels, val_labels = train_test_split(train_images, train_labels, test_size=0.2, shuffle=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3d3b850",
+   "metadata": {},
+   "source": [
+    "### Implement ResNet-18 model\n",
+    "---\n",
+    "Implement the resnet18 model for CIFAR10 dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "659f2f72",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class BasicBlock(Model):\n",
```

Comment:
I think you can just use the existing reset implementation in `tf.keras.applications` instead of implementing it your self.